app/code/community/Mbiz/Reports/Model/Sales/Journal/Result/CreditMemo.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * This file is part of MBiz_Reports for Magento.
+ *
+ * @license All rights reserved
+ * @author Mina Amrouche <m.amrouche@monsieurbiz.com> <@mina_amrouche>
+ * @category MBiz
+ * @package MBiz_Reports
+ * @copyright Copyright (c) 2017 Monsieur Biz (https://monsieurbiz.com/)
+ */
+
+/**
+ * Sales_Journal_Result_CreditMemo Model
+ * @package MBiz_Reports
+ */
+class MBiz_Reports_Model_Sales_Journal_Result_CreditMemo extends Mbiz_Reports_Model_Sales_Journal_Result_Abstract
+{
+
+// Monsieur Biz Tag NEW_CONST
+
+// Monsieur Biz Tag NEW_VAR
+
+    protected function _load(\Mbiz_Reports_Model_Sales_Journal_Request $request, \Zend_Date $from, \Zend_Date $to)
+    {
+        $read = $this->getReadAdapter();
+        $select = $read->select();
+        $select
+            ->from([
+                'memo' => $this->getTableName('sales/creditmemo')
+            ], [
+                'total_credit_memo' => 'SUM(ROUND(memo.grand_total, 2))',
+            ])
+            ->where("DATE(memo.created_at) >= ?", $from->toString('y-MM-dd'))
+            ->where("DATE(memo.created_at) <= ?", $to->toString('y-MM-dd'))
+        ;
```

Comment:
Is it including or excluding taxes?

Btw, what about taxes? Maybe it can be shown in the grid too. WDYT?